### PR TITLE
[Core] Use the exception handler in the inspector

### PR DESCRIPTION
**Pull request by orta**
So that when you don't have a specific message, the query is simplified around the exception - this should allow it to cast a wider net.

### CODE REVIEW

File: fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb [hunk 1]
```diff
@@ -94,7 +95,11 @@ def show_github_issues(message)
 
       inspector = GhInspector::Inspector.new("fastlane", "fastlane", verbose: $verbose)
       delegate = Fastlane::InspectorReporter.new
-      inspector.search_query(message, delegate)
+      if message_or_error.is_a? String
```
Comment (KrauseFx) [inline fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb]: Missing `(` around `String` 🚎

Comment (KrauseFx) [inline fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb]: :trollface:

**Comment by KrauseFx** [comment]
Looks great, however the tests are failing:
```
lib/fastlane_core/ui/fastlane_runner.rb:98:27: C: Prefer Object#kind_of? over Object#is_a?.
      if message_or_error.is_a? String
```

**Comment by orta** [comment]
Ah, good old rubocop, I'm alright with that change 👍 

and yer troll-y `(` and `)`'s

**Comment by orta** [comment]
Pushed an amended commit 👍

File: fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb [hunk 2]
```diff
@@ -94,7 +95,11 @@ def show_github_issues(message)
 
       inspector = GhInspector::Inspector.new("fastlane", "fastlane", verbose: $verbose)
       delegate = Fastlane::InspectorReporter.new
-      inspector.search_query(message, delegate)
+      if message_or_error.kind_of?(String)
```
Comment (KrauseFx) [inline fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb]: ❤️

**Comment by KrauseFx** [comment]
This is great, thanks 👍

**Pull request merged**